api/src/main/java/marquez/db/SourceDao.java

```diff
@@ -69,19 +47,29 @@
           + "created_at, "
           + "updated_at, "
           + "name, "
-          + "connection_url "
+          + "connection_url, "
+          + "description "
           + ") VALUES ("
           + ":uuid, "
           + ":type, "
-          + ":now, "
-          + ":now, "
+          + ":createdAt, "
+          + ":updatedAt, "
```

Comment:
Why add a createdAt field? The only instance I can think of is if you want to create a record and set an updated time to be different than when you created it. That doesn't seem like a valid use case to me.